#goal         
float64 dist_from_tag
float64 tagPosY
float64 des_ang
int64 marker
---
#result
float64 th_fin
float64 x_fin
---
#feedback
int64 fase